tide/item.go

```diff
@@ -109,12 +109,11 @@ type PhpcsSummary struct {
 type LighthouseResults struct{}
 
 type LighthouseSummary struct {
-	ReportCategories []LighthouseCategory `json:"reportCategories,omitempty"`
+	Categories map[string]LighthouseCategory `json:"categories,omitempty"`
 }
 
 type LighthouseCategory struct {
-	Name        string  `json:"name"`
-	Weight      float32 `json:"weight"`
+	Title       string  `json:"title"`
 	Description string  `json:"description"`
 	Id          string  `json:"id"`
```

Comment:
@valendesigns is this the key that's been removed from the report? If so, we could either `omitempty` or remove it completely.

Reply:
`weight` was removed it's just that `id` is redundant now that the array key is the `id`. But not a big deal.